### PR TITLE
test

### PR DESCRIPTION
**Pull request by guilherme-andrade**
changed some h1

### CODE REVIEW

**Comment by guilherme-andrade** [comment]
elvnljekvhlvtnçkweglj

File: app/views/shared/_navbar.html.erb [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <%= link_to root_path, class: "navbar-wagon-brand" do %>
     <%= image_tag "logo.png" %>
   <% end %>
-
+  <h1>dasdasd</h1>
```
Comment (turohakulinen) [inline app/views/shared/_navbar.html.erb]: not good

Comment (turohakulinen) [reply app/views/shared/_navbar.html.erb]: mayby

**Review by turohakulinen — COMMENTED** [review]
oh

**Pull request merged**